.github/workflows/haskell.yml

```diff
@@ -62,22 +62,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.8", "9.6.6", "9.8.2", "9.10.1", "9.12.1"]
+        ghc: ["9.6", "9.8", "9.10", "9.12"]
         os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
-        # GHC versions older than ghc-9.2 are not supported on macos-latest
+        # GHC versions older than ghc-9.6 are not supported on macos-latest
```

Comment:
Why do you think GHC versions older that ghc-9.6 all of a sudden stopped being supported by `macos-latest`?
If you read this comment carefully, you will realize that all of those `exclude`s can be removed, because they were here only because: "GHC versions older than ghc-9.2 are not supported on macos-latest"
So, please remove this whole section,

Reply:
This is a grave oversight. We don't need to exclude anything now. :+1: